lib/datastore/query.js

```diff
@@ -75,7 +75,15 @@ function Query(namespace, kinds) {
 }
 
 /**
+ * @param {boolean=} autoPaginateVal - Have pagination handled automatically.
+ *     Default: true.
  * @return {module:datastore/query}
+ *
+ * @example
+ * // List all people related to person 1234 with a single query
+ * var query = dataset.createQuery('Person')
+ *   .hasAncestor(dataset.key(['Person', 1234]))
+ *   .autoPaginate();
```

Comment:
Since this is the default, we should show running a query as false, since anyone who comes to look at this doc is probably wondering how to shut it off.